src/NuGet.Core/NuGet.Frameworks/FallbackFramework.cs

```diff
@@ -6,11 +8,11 @@ namespace NuGet.Frameworks
     public class FallbackFramework : NuGetFramework, IEquatable<FallbackFramework>
     {
         /// <summary>
-        /// Secondary framework to fall back to.
+        /// List of frameworks to fall back to.
         /// </summary>
-        public NuGetFramework Fallback { get; }
+        public IEnumerable<NuGetFramework> Fallback { get; }
 
-        public FallbackFramework(NuGetFramework framework, NuGetFramework fallbackFramework)
+        public FallbackFramework(NuGetFramework framework, IEnumerable<NuGetFramework> fallbackFramework)
```

Comment:
Throw if the fallbackFramework list is empty, if it is this shouldn't be a FallbackFramework.